app/views/replies/_reply.html.erb

```diff
@@ -42,9 +52,11 @@
 
       <% if user_signed_in? %>
         <% if reply.user.id == current_user.id %>
-          <%= link_to "Delete", {:action => 'destroy', :controller => "replies", :id => @reply.id, :new_thread_id => @new_thread.id}, :method => "delete", data: { confirm: 'Are you sure?' }, :class => 'reply-link edit-post'%>
+          <a id="edit-reply-btn-<%= reply.id %>" class="btn btn-default" reply-id="<%= reply.id %>" onClick="toggleEdit(this)"><i class="fa fa-pencil"></i></a>
```

Comment:
I have associated an onClick event that takes care of the toggling. The `reply-id` attribute stores the id of the reply which is used in the `toggleEdit()` function.